week-2/implement/bmi.js

```diff
@@ -13,3 +13,21 @@
 // Given someone's weight in kg and height in metres
 // Then when we call this function with the weight and height
 // It should return their Body Mass Index to 1 decimal place
+
+function calcularIMC(peso, altura) {
```

Comment:
This code works, however just a reminder. CYF is about making you employable in the UK and so it's important to try to use English in your code. It may not always be easy but it's important practice for the real world.